doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,543 @@
+    libswscale-dev \
+    libavresample-dev \
+    libtbb2 \
+    libssl-dev \
+    libva-dev \
+    libmfx-dev \
+    libgstreamer1.0-dev \
+    libgstreamer-plugins-base1.0-dev
+    @endcode
+
+### Procedure {#tutorial_openvino_install_building_on_ubuntu_procedure}
+
+1.  Copy OpenCV repository:
+    @code{.bash}
+	git clone --recurse-submodules https://github.com/opencv/opencv.git
+	@endcode
+
+2.  Create build directory and enter into it:
+    @code{.bash}
+    mkdir ~/build-opencv && cd ~/build-opencv
+    @endcode
+
+3.  Compile and install OpenCV:
+    @code{.bash}
+	cmake -G Ninja \
+          -D BUILD_INFO_SKIP_EXTRA_MODULES=ON \
+          -D BUILD_EXAMPLES=OFF \
+          -D BUILD_JASPER=OFF \
+          -D BUILD_JAVA=OFF \
+          -D BUILD_JPEG=ON \
+          -D BUILD_APPS_LIST=version \
+          -D BUILD_opencv_apps=ON \
+          -D BUILD_opencv_java=OFF \
+          -D BUILD_OPENEXR=OFF \
+          -D BUILD_PNG=ON \
+          -D BUILD_TBB=OFF \
+          -D BUILD_WEBP=OFF \
+          -D BUILD_ZLIB=ON \
+          -D WITH_1394=OFF \
+          -D WITH_CUDA=OFF \
+          -D WITH_EIGEN=OFF \
+          -D WITH_GPHOTO2=OFF \
+          -D WITH_GSTREAMER=ON \
+          -D OPENCV_GAPI_GSTREAMER=OFF \
+          -D WITH_GTK_2_X=OFF \
+          -D WITH_IPP=ON \
+          -D WITH_JASPER=OFF \
+          -D WITH_LAPACK=OFF \
+          -D WITH_MATLAB=OFF \
+          -D WITH_MFX=ON \
+          -D WITH_OPENCLAMDBLAS=OFF \
+          -D WITH_OPENCLAMDFFT=OFF \
+          -D WITH_OPENEXR=OFF \
+          -D WITH_OPENJPEG=OFF \
+          -D WITH_QUIRC=OFF \
+          -D WITH_TBB=OFF \
+          -D WITH_TIFF=OFF \
+          -D WITH_VTK=OFF \
+          -D WITH_WEBP=OFF \
+          -D CMAKE_USE_RELATIVE_PATHS=ON \
+          -D CMAKE_SKIP_INSTALL_RPATH=ON \
+          -D ENABLE_BUILD_HARDENING=ON \
+          -D ENABLE_CONFIG_VERIFICATION=ON \
+          -D ENABLE_PRECOMPILED_HEADERS=OFF \
+          -D ENABLE_CXX11=ON \
+          -D INSTALL_PDB=ON \
+          -D INSTALL_TESTS=ON \
+          -D INSTALL_C_EXAMPLES=ON \
+          -D INSTALL_PYTHON_EXAMPLES=ON \
+          -D CMAKE_INSTALL_PREFIX=install \
+          -D OPENCV_SKIP_PKGCONFIG_GENERATION=ON \
+          -D OPENCV_SKIP_PYTHON_LOADER=OFF \
+          -D OPENCV_SKIP_CMAKE_ROOT_CONFIG=ON \
+          -D OPENCV_GENERATE_SETUPVARS=OFF \
+          -D OPENCV_BIN_INSTALL_PATH=bin \
+          -D OPENCV_INCLUDE_INSTALL_PATH=include \
+          -D OPENCV_LIB_INSTALL_PATH=lib \
+          -D OPENCV_CONFIG_INSTALL_PATH=cmake \
+          -D OPENCV_3P_LIB_INSTALL_PATH=3rdparty \
+          -D OPENCV_SAMPLES_SRC_INSTALL_PATH=samples \
+          -D OPENCV_DOC_INSTALL_PATH=doc \
+          -D OPENCV_OTHER_INSTALL_PATH=etc \
+          -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
+          -D OPENCV_INSTALL_FFMPEG_DOWNLOAD_SCRIPT=ON \
+          -D BUILD_opencv_world=OFF \
+          -D BUILD_opencv_python2=OFF \
+          -D BUILD_opencv_python3=ON \
```

Comment:
This option requires at least python3, python3-dev packages installed + numpy

Reply:
Fixed